earthnet_models_pytorch/model/convlstm_ae.py

```diff
@@ -374,25 +404,35 @@ def forward(
             # Copy the previous prediction for skip connections
             pred_previous = torch.clone(pred)
 
-            
-
             # Teacher forcing scheduled sampling
             if (
                 self.hparams.teacher_forcing
                 and self.training
                 and torch.bernoulli(teacher_forcing_decay)
             ):
-                pred = target[:, t, 0, ...].unsqueeze(1)
+                pred = target[
+                    :,
+                    t,
+                    : (
+                        self.hparams.num_outputs
+                        if not self.hparams.decoder_input_subtract_s2bands
+                        else 1
```

Comment:
why not else 0?

Reply:
`arr[:1] == arr[0].unsqueeze(0)` <-- thats why

Reply:
ok thanks! :D